### PR TITLE
Create git utils and test

### PR DESCRIPTION
**Pull request by Chiiizzzy**
Signed-off-by: Kaiyuan Hu <kaiyuan.hu@zilliz.com>

### CODE REVIEW

**Review by shiyu22 — DISMISSED** [review]

File: towhee/hub/repo_manager.py [hunk 1]
```diff
@@ -40,6 +38,8 @@ class RepoManager:
             The author of the repo.
         repo (`str`):
             The name of the repo.
+        root (`str`):
```
Comment (shiyu22) [inline towhee/hub/repo_manager.py:41]: Can you ignore this file, and there is another PR #484  for it.

Comment (Chiiizzzy) [reply towhee/hub/repo_manager.py:41]: We can just leave it here and keep your change when resolve conflicts

Comment (shiyu22) [reply towhee/hub/repo_manager.py:41]: OK

**Review by shiyu22 — APPROVED** [review]

File: towhee/utils/git_utils.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        author (`str`):
+            The author of the repo.
+        repo (`str`):
+            The name of the repo.
+        root (`str`):
+            The root url of the repo.
+        self._repo_path (`Union[str, Path]`):
+            The path to the local repo.
+    """
+    def __init__(self, author: str, repo: str, root: str = 'https://hub.towhee.io', repo_path: Union[str, Path] = None):
```
Comment (junjiejiangjjj) [inline towhee/utils/git_utils.py:40]: The `repo_path` maybe named `local_repo_path`

File: towhee/utils/git_utils.py [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+        """
+        A wrapper function for git commit.
+
+        Commit current changes in the repo.
+
+        Args:
+            cmt_msg (`str`):
+                The commit message.
+        """
+        if isinstance(self._repo_path, str):
```
Comment (junjiejiangjjj) [inline towhee/utils/git_utils.py:156]: What does `self._repo_path`  do here?

File: towhee/utils/git_utils.py (hunk 2 above)
Comment (junjiejiangjjj) [reply towhee/utils/git_utils.py:40]: Overview of the code, it seems only `clone` needs `repo_path`

Comment (Chiiizzzy) [reply towhee/utils/git_utils.py]: Fixed

File: towhee/utils/git_utils.py (hunk 3 above)
Comment (Chiiizzzy) [reply towhee/utils/git_utils.py]: Fixed

**Pull request merged**